crates/core/component/shielded-pool/Cargo.toml

```diff
@@ -50,7 +50,7 @@ decaf377-ka = {workspace = true}
 decaf377-rdsa = {workspace = true}
 futures = {workspace = true}
 hex = {workspace = true}
-ibc-proto = {workspace = true, default-features = false}
+ibc-proto = {workspace = true, features = ["transport"], default-features = false}
```

Comment:
This is to expose the generated grpc services of cosmos-sdk; the `transport` feature now sets `cosmos-sdk-proto/grpc-transport`, which was previously always exported.